pkg/monitoring/metrics/forklift-controller/metrics.go

```diff
@@ -0,0 +1,117 @@
+package forklift_controller
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+const (
+	Succeeded = "Succeeded"
+	Failed    = "Failed"
+	Executing = "Executing"
+	Running   = "Running"
+	Pending   = "Pending"
+	Canceled  = "Canceled"
+	Blocked   = "Blocked"
+	Ready     = "Ready"
+	Deleted   = "Deleted"
+	Warm      = "Warm"
+	Cold      = "Cold"
+	Local     = "Local"
+	Remote    = "Remote"
+)
+
+var (
+	// 'status' - [ succeeded, failed, Executing, Canceled]
+	// 'provider' - [oVirt, vSphere, Openstack, OVA, openshift]
+	// 'mode' - [Cold, Warm]
+	// 'target' - [Local, Remote]
+	migratioStatusCounter = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "mtv_migrations_status",
```

Comment:
based on the `Help` below, maybe "mtv_migrations"? as we don't show only the status.

Reply:
its in telemetry terms, it not refereeing only to the status of the migration but the status of the object.